From 57762b98cb8390c8abcc918dc342588bfe8b3d3c Mon Sep 17 00:00:00 2001
From: taaem <taaem@mailbox.org>
Date: Sat, 10 Dec 2016 14:06:14 +0100
Subject: [PATCH 2/2] Only build for x64 Disable 32bit builds because they
 aren't necessary for Solus which only uses x64. Also 32bit build breaks the
 whole build process.

---
 Gruntfile.coffee | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Gruntfile.coffee b/Gruntfile.coffee
index 820a9f7..345fe25 100755
--- a/Gruntfile.coffee
+++ b/Gruntfile.coffee
@@ -259,5 +259,5 @@ module.exports = (grunt) ->
   grunt.registerTask 'win',        ['clean:win', 'update-keys', 'release-internal', 'electron:win_internal', 'create-windows-installer:internal']
   grunt.registerTask 'win-prod',   ['clean:win', 'update-keys', 'release-prod', 'electron:win_prod', 'create-windows-installer:prod']
 
-  grunt.registerTask 'linux',      ['clean:linux', 'update-keys', 'release-internal', 'shell']
-  grunt.registerTask 'linux-prod', ['clean:linux', 'update-keys', 'release-prod', 'shell']
+  grunt.registerTask 'linux',      ['clean:linux', 'update-keys', 'release-internal', 'shell:linux64']
+  grunt.registerTask 'linux-prod', ['clean:linux', 'update-keys', 'release-prod', 'shell:linux64']
-- 
2.11.0

